staging/src/k8s.io/apiserver/pkg/server/filters/maxinflight.go

```diff
@@ -94,6 +97,7 @@ func WithMaxInFlightLimit(
 				defer func() { <-c }()
 				handler.ServeHTTP(w, r)
 			default:
+				metrics.MonitorRequest(r, strings.ToUpper(requestInfo.Verb), requestInfo.Resource, requestInfo.Subresource, "", errors.StatusTooManyRequests, time.Now())
```

Comment:
I would prefer a ToUpper inside of MonitorRequest to be immune to wrong case. Too easy to get this wrong from the outside.